### ISSUE TITLE
Reviewer Id is empty when the $body is constructed.

### ISSUE THREAD

**Issue by DamienGaskell** [shayki5/azure-devops-create-pr-task#149]
This task has recently changed so that when my entry in the "Reviewer" is submitted for a pull request, the following error is shown. The console output that shows the Json structure - the $Id is empty and the $IsReq is assigned OK.

Error = ##[error]{"$id":"1","innerException":null,"message":"Invalid argument value.\r\nParameter name: The identity  is not recognized.","typeName":"Microsoft.TeamFoundation.SourceControl.WebServer.InvalidArgumentValueException, Microsoft.TeamFoundation.SourceControl.WebServer","typeKey":"InvalidArgumentValueException","errorCode":0,"eventId":0}

The output log is attached. I tried many combinations of fields, but all failed. If I leave the Reviewers blank, it will work.
This task is used in the production environment and is causing great problems for the system management. Can the release of new features and fixes be staged in the releases so that they can be UAT tested first?

[tasklog_6 (TestPulls).log](https://github.com/shayki5/azure-devops-create-pr-task/files/9565912/tasklog_6.TestPulls.log)

**Comment by shayki5** [comment]
Hi @DamienGaskell, is it worked for you in the past with the same reviewers? 
Can you please run the pipeline with `system.debug` = `true` and share the log?

**Comment by DamienGaskell** [comment]
This began to fail with the same settings, same reviewers, same everything.
I cannot claim that there have not been any changes to the internal settings of the reviewers in Devops by other administrators, but nothing I know about has changed.

[tasklog_6 (pulltest1).log](https://github.com/shayki5/azure-devops-create-pr-task/files/9570728/tasklog_6.pulltest1.log)

**Comment by DamienGaskell** [comment]
I have looked back at some older jobs. This was working for version 1.2.335

**Comment by shayki5** [comment]
The version now is `1.2.335`... are you sure is it worked?

**Comment by shayki5** [comment]
If you run this api:
`https://ftrintegration.vsaex.visualstudio.com/_apis/userentitlements?top=10000&api-version=4.1-preview.1`
Do you get `damien.gaskell@frontiersoftware.com.au` in the results?

**Comment by DamienGaskell** [comment]
Hi,
you have done it. The email returned from the URI is not the same as the one I used in the Referrers. We have an additional email for the Azure DevOps. It was working before with the referrer I used, but something must have changed in the background.
I changed the referrer to damien.gaskell@frontiersoftwareADO.com and it is working again.

**Thank You** for your time and your immediate responses. Your app is well behaved and has streamlined the system development without having to reinvent the wheel.
